frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -178,6 +178,7 @@ export interface PipelineRun extends K8sResourceKind {
     startTime?: string;
     completionTime?: string;
     taskRuns?: PLRTaskRuns;
+    pipelineSpec?: PipelineSpec;
```

Comment:
```suggestion
    pipelineSpec: PipelineSpec;
```
Based on my review comment.